ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/websocket/subscription/SubscriptionManager.java

```diff
@@ -49,6 +49,15 @@
   private final Map<Long, Subscription> subscriptions = new HashMap<>();
   private final Map<String, List<Long>> connectionSubscriptionsMap = new HashMap<>();
   private final SubscriptionBuilder subscriptionBuilder = new SubscriptionBuilder();
+  private final long refreshDelay;
+
+  public SubscriptionManager(final long refreshDelay) {
+    this.refreshDelay = refreshDelay;
+  }
+
+  public SubscriptionManager() {
+    this.refreshDelay = 5000;
```

Comment:
use the const here rather than the number

Reply:
Fixed